e2etests/testcase_test.go

```diff
@@ -1204,12 +1204,13 @@ var testCases = []testCase{
 									"Score":               4.3,
 									"Vectors":             "AV:N/AC:M/Au:N/C:N/I:N/A:P",
 								},
-								"CVSSv3": map[string]interface{}{
-									"ExploitabilityScore": 2.8,
-									"ImpactScore":         3.6,
-									"Score":               6.5,
-									"Vectors":             "CVSS:3.1/AV:N/AC:L/PR:N/UI:R/S:U/C:N/I:N/A:H",
-								},
+								// NVD stopped returning this
```

Comment:
Can you create a ticket to track this?

Reply:
I may instead change the sanity test to fail if this data comes back so that we will not need a ticket and can rely on CI failures instead.

Reply:
I modified the tests to fail if the data comes back in the future.

Reply:
SGTM